### PR TITLE
CLOUDP-82263 user can use helm chart

### PR DESCRIPTION
**Pull request by leo-ri**

### CODE REVIEW

File: test/e2e/cli/helm/helm.go [hunk 1]
```diff
@@ -32,8 +35,137 @@ func InstallTestApplication(input model.UserInputs, user model.DBUser, port stri
...
 		"test-app-"+user.Spec.Username,
 		config.HELMTestAppPath,
-		"--set-string", fmt.Sprintf("connectionSecret=%s-%s-%s", input.ProjectName, input.Clusters[0].Spec.Name, user.Spec.Username),
+		"--set-string", fmt.Sprintf("connectionSecret=%s-%s-%s", input.Project.GetProjectName(), input.Clusters[0].Spec.Name, user.Spec.Username),
 		"--set-string", fmt.Sprintf("nodePort=%s", port),
 		"-n", input.Namespace,
 	)
 }
+
+func InstallCRDToNamespace(input model.UserInputs) {
```
Comment (antonlisovenko) [inline test/e2e/cli/helm/helm.go:44]: [q] is there a difference between this func and the one on line 53? CRDs don't reference any namespaces so I guess the end result should  be the same?

File: test/e2e/helm_chart_test.go [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+			userSpec.Clusters[0].Spec.Name = "cluster-from-helm"
+		})
+		By("User use helm for deploying operator", func() {
+			helm.AddMongoDBRepo()
+			helm.InstallKubernetesOperatorNS(userSpec)
+		})
+		deployCluster(userSpec, 30006)
+	})
+
+	It("[helm-wide] User can deploy operator namespaces by using HELM", func() {
```
Comment (antonlisovenko) [inline test/e2e/helm_chart_test.go:68]: what's the difference between this test case and the https://github.com/mongodb/mongodb-atlas-kubernetes/pull/204/files#diff-1c7d404bffb0725622f7419f40cd991c5765794f024aec084ab5d5fc5b3f6fafR45 ? I see they install the same helm charts and the same clusters?

File: test/e2e/helm_chart_test.go [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+		Eventually(
+			func() bool {
+				return mongocli.IsProjectInfoExist(userSpec.ProjectID)
+			},
+			"5m", "20s",
+		).Should(BeFalse(), "Project and cluster should be deleted from Atlas")
+	})
+
+	By("Delete HELM release", func() {
+		helm.UninstallKubernetesOperatorNS(userSpec)
```
Comment (antonlisovenko) [inline test/e2e/helm_chart_test.go:132]: Can we check that the resources in Atlas were removed eventually?

**Review by antonlisovenko — APPROVED** [review]
LGTM once the tests are green, helm is installed in clusterwide/namespaced and additional check for helm delete
🥇

File: test/e2e/helm_chart_test.go (hunk 2 above)
Comment (leo-ri) [reply test/e2e/helm_chart_test.go:68]: you're right , i'll change the call

File: test/e2e/helm_chart_test.go (hunk 3 above)
Comment (leo-ri) [reply test/e2e/helm_chart_test.go]: The operator itself? in Atlas, we check cluster/project deletion

**Pull request merged**